packages/sku/lib/configure.js

```diff
@@ -109,4 +113,24 @@ module.exports = async () => {
     comment: 'managed by sku',
     patterns: gitIgnorePatterns.map(convertToForwardSlashPaths),
   });
+
+  // Check for `.less` files
+  const lessFileGlobResults = await Promise.all(
+    paths.src
+      .filter((srcPath) => fs.statSync(srcPath).isDirectory())
```

Comment:
For whatever reason, async `fs.stat` throws an exception on certain files, like `sku.config.ts`. I don't know why.